joints_detectors/Alphapose/fn.py

```diff
@@ -194,7 +194,7 @@ def vis_frame(frame, im_res, format='coco'):
             bg = img.copy()
             cv2.circle(bg, (int(cor_x / 2), int(cor_y / 2)), 2, p_color[n], -1)
             # Now create a mask of logo and create its inverse mask also
-            transparency = max(0, min(1, kp_scores[n]))
+            transparency = max(0, min(1, kp_scores[n])).item()
```

Comment:
`max(0, min(1, kp_scores[n].item()))` is better, since the int value (0 or 1) does not have `.item()` function in edge cases.